pkg/osutil/osutil_unix.go

```diff
@@ -93,43 +93,3 @@ func LongPipe() (io.ReadCloser, io.WriteCloser, error) {
 	prolongPipe(r, w)
 	return r, w, err
 }
-
-// ProcessExitStatus returns process exit status.
-// This is here only because of fuchsia that does not implement WaitStatus.
-func ProcessExitStatus(ps *os.ProcessState) int {
-	return ps.Sys().(syscall.WaitStatus).ExitStatus()
-}
-
-// CreateMemMappedFile creates a temp file with the requested size and maps it into memory.
-func CreateMemMappedFile(size int) (f *os.File, mem []byte, err error) {
```

Comment:
If we don't use the method anywhere, we should also delete it from other `osutil_OS.go` implementations.

Reply:
Thanks, done.